xml/cap_admin_backup-restore.xml

```diff
@@ -521,5 +523,249 @@
+      </listitem>
+     </itemizedlist>
+    </step>
+   </procedure>
+  </sect2>
+  <sect2 xml:id="sec.cap.raw-data-restore-procedure">
+   <title>Performing a raw data restore</title>
+   <para>
+    Perform the following steps to restore your backed up data to the target SCF deployment.
+   </para>
+   <important>
+    <title>Ensure access to the correct SCF deployment</title>
+     <para>
+      Working with multiple &kube; clusters simultaneously can be confusing. Ensure you're communicating with the desired cluster <link xlink:href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#set-the-kubeconfig-environment-variable">by setting <literal>$KUBECONFIG</literal> correctly</link>.
+     </para>
+   </important>
+   <procedure>
+    <step>
+     <para>
+      The target SCF cluster needs to be deployed with the correct database encryption key(s) set in your <filename>scf-config-values.yaml</filename> before data can be restored. How the encryption key(s) will be prepared in your <filename>scf-config-values.yaml</filename> depends on the result of <xref linkend="get-encryption-key"/> in <xref linkend="sec.cap.raw-data-backup-procedure"/>
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        If <literal>current_key_label</literal> was set, use the <literal>current_key_label</literal>
+        obtained as the value of <literal>CC_DB_CURRENT_KEY_LABEL</literal> and all the keys under the
+        <literal>keys</literal> are defined under <literal>CC_DB_ENCRYPTION_KEYS</literal>.
+        See the following example <filename>scf-config-values.yaml</filename>:
+       </para>
+<screen>env:
+  CC_DB_CURRENT_KEY_LABEL: <replaceable>migrated_key_1</replaceable>
+
+secrets:
+  CC_DB_ENCRYPTION_KEYS:
+    <replaceable>migrated_key_1</replaceable>: "<replaceable>&lt;key_goes_here></replaceable>"
+    <replaceable>migrated_key_2</replaceable>: "<replaceable>&lt;key_goes_here></replaceable>"
+</screen>
+      </listitem>
+      <listitem>
+       <para>
+        If <literal>current_key_label</literal> was not set, create one for the new cluster
+	through <filename>scf-config-values.yaml</filename> and set it to the
+	<literal>$DB_ENCRYPTION_KEY</literal> value from the old cluster.In this example,
+	<literal>migrated_key</literal> is the new <literal>current_key_label</literal> created:
+       </para>
+<screen>env:
+  CC_DB_CURRENT_KEY_LABEL: <replaceable>migrated_key</replaceable>
+
+secrets:
+  CC_DB_ENCRYPTION_KEYS:
+  <replaceable>migrated_key</replaceable>: "<replaceable>OLD_CLUSTER_DB_ENCRYPTION_KEY</replaceable>"
+</screen>
+      </listitem>
+     </itemizedlist>
+    </step>
+    <step>
+     <para>
+       Deploy a non-high-availability configuration of SCF (see <xref linkend="sec.cap.install-scf-prod"/>) and waiting until all pods are ready before proceeding.
+     </para>
```

Comment:
wait, not waiting

Reply:
Addressed in https://github.com/SUSE/doc-cap/pull/280/commits/fcde67d3da1a19b5743da5d0fb77df36f5e37f01